Makefile

```diff
@@ -127,8 +127,9 @@ KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path $(KUBEBUILD
 
 .PHONY: test
 test: $(SETUP_ENVTEST) $(GOVC)
+	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" GOVC_BIN_PATH=$(GOVC) go test -v ./apis/... ./controllers/... ./pkg/... $(TEST_ARGS)
```

Comment:
Removed `;` after GOVC_BIN_PATH, if we export the variables. It would be set across the shell and would not be the right way. SO instead, setting it for this particular instance.